### ISSUE TITLE
World unplayable due to disconnects within 30-60 seconds of login

### ISSUE THREAD

**Issue by Silverlyning** [StrangeLoopGames/EcoIssues#7996]
We've been experiencing more and more issues with disconnects lately. In fact, we recently reached the point where we were unable to remain logged in for more than 30-60 seconds. This problem affects all of our world's players.

We tried reverting to an earlier world save file, and that helped for a day, but it's once again unplayable. Prior to Beta 0.7.4.6 we never used to have these issues. 

We are on day 26 of a 45 day meteor event. Our world runs 24/7 on a dedicated home computer (Windows 10 Pro 64-Bit) with ample memory and storage space. It's a standalone Eco Server (non-Steam). I reboot (and backup) the server computer every 2-3 days, so our downtime is very low. Rebooting the server computer does not fix the issue. The world is currently down due to this issue. There are only 6 players and the world is 140x140, so it's got quite a low player density. This is our second world, and we never had any issues like this on our first world!

I'm not sure if it's relevant, but I noticed that the problems seem to occur once we began adding Wind Turbines to our world. Unfortunately removing them doesn't fix the issue. 

Our world is literally unplayable now! Any ideas what might be causing these problems? Is there anything we can try to fix the issue? 

Here's a link to the world (Happyworld2) files, along with settings and logs: 
https://drive.google.com/open?id=1dD-DExBrwFwCQ5LbVdAHlg_-3hIOpWiM

**Comment by metachronism** [comment]
@Silverlyning I was not able to reproduce your issue with the latest staging version (7.5+)

could you try that version (backup your save etc ahead of time) and see if that works for you?  7.5 should be out this week

**Comment by Silverlyning** [comment]
I tried to use your Server software from the Staging version, but had to download it from the Bleeding Edge – Beta 7.5.0 #051d6d32, since the Staging area was still showing Beta 0.7.4.0 #518a927f on your downloads page. I updated the server software and then moved my Configs and Storage folders to the new Server folder.  After that, I started up the new Server (7.5.0) software on my Server PC.

I then used the Steam Beta tab to update Eco (on my Games PC) to “staging - 7.5 Staging”. After the update was applied I tried to join my world, but it seems like there might be a version conflict, because there was no option to join the world. The button in the lower right corner (of the Join World screen) shows the version number and is not clickable. Are your two versions of Eco 0.7.5 (Stand-alone Server and Steam Game) compatible?

**Comment by Silverlyning** [comment]
With the official update to Beta v0.7.5, I updated our server, and again tried to use our world "Happyworld2". This time there was no version mismatch that I had experienced with the Staging software. 

I was able to join the game, however the disconnect occurred within 30-60 seconds of me logging in. Also, there is a problem with the terrain drawing now as well. Walking onto the invisible land doesn't trigger the draw routine either. 

![happyworld2-20180626](https://user-images.githubusercontent.com/40070019/41951978-865015ba-7982-11e8-8eae-463d91bfe052.png)
